code/datums/actions/item_action.dm

```diff
@@ -6,6 +6,8 @@
 	 * e.g. gun attachment action: target = attachment, holder = gun.
 	 */
 	var/obj/item/holder_item
+	/// Defines wheter we overlay the image of the obj we are linked to
+	var/get_obj_image = TRUE
```

Comment:
I'd consider renaming this to something like "use_obj_appearance" since that explains what the var toggles better